src/shared/watchdog.c

```diff
@@ -173,6 +150,11 @@ int watchdog_setup(usec_t timeout) {
         if (watchdog_fd >= 0 && watchdog_timeout == timeout)
```

Comment:
This function call really should have some comment here what timeout == 0 and timeout == USEC_INFINITY actually are supposed to mean

Reply:
Done.